apps/token/src/routes/staking/home/staking-intro.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { StakingIntro } from './staking-intro';
+
+describe('Staking', () => {
```

Comment:
suggestion (non blocking): I find little value in this test, no harm leaving it in but I'd suggest not bothering in future